requirements/pip-tools.txt

```diff
@@ -4,6 +4,10 @@
 #
 #    make upgrade
 #
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
+--extra-index-url https://pypi.python.org/simple
+--trusted-host edx.devstack.devpi
+
```

Comment:
Please remove these. We want to be able to create a virtualenv outside of the devstack. I deleted these intentionally in https://github.com/open-craft/section-to-course/pull/1/commits/465e723a7ba4b09ae209bfb05f2cb5b7f65ca5c1.

Reply:
Ah-- these were added automatically when I ran `make upgrade` from within the devstack. I've removed them.